config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -91,3 +91,46 @@ periodics:
     testgrid-num-failures-to-alert: '1'
     testgrid-dashboards: sig-security-snyk-scan
     description: Run snyk scan on k/k master periodically
+- name: auto-refreshing-official-cve-feed
+  interval: 2h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+  labels:
+    preset-service-account: "true" #TODO
+  spec:
+    serviceAccountName: k8s-cve-feed #TODO
+    containers:
+    - #image: gcr.io/k8s-staging-test-infra/bazelbuild:v20210930-b6c4097796-test-infra
+      image: python3.7 #TODO
+      command:
+      - hack/fetch-cve-feed.sh
+      env:
+      - name: CVE_GCS_PATH
+        value: "gs://k8s-cve-feed"
+      - name: CVE_GCS_PROJECT
+        value: "k8s-infra-prow-build-trusted" #TODO
```

Comment:
you don't need to add a project in you want to interact (PUT,HEAD) with a GCS bucket.

Reply:
Updated